provision/minikube/README.adoc

```diff
@@ -140,8 +140,9 @@ The setup is as follows:
 * https://www.jaegertracing.io/[Jaeger] collects traces from Keycloak running inside Minikube, and can also receive traces from a locally running test application.
 * https://cryostat.io/[Cryostat] can connect to Keycloak instances and create Java Flight Recorder (JFR) recordings.
 
-Logs, traces and metrics are stored within Minikube for 7 days.
-This can be overwritten by adding in an `.env` file with the setting `KB_RETENTION=XXd` to change it to a different value.
+Logs, traces and metrics are stored within Minikube for 7 days = 168 hours.
```

Comment:
would it be possible to set the least minimum retention to be 1 hour ?

Reply:
or even decimal hours ?

Reply:
This setting applies to Jaeger, Prometheus and Loki. While each of them parsing this a little differently, the common denominator seems to be hours. I didn't test decimal hours, and I wouldn't recommend to do so. 

For Loki I remember that the minimum retention period is 24 hours, see https://grafana.com/docs/loki/latest/operations/storage/retention/

This is what I recommend: Once the problem with the nightly operator is fixed, merge the PR. We could leave the original issue open and I'll create a new PR with additional documentation about retention times and how to test this.